controllers/machine_controller.go

```diff
@@ -537,18 +538,18 @@ func (r *MachineReconciler) drainNode(ctx context.Context, cluster *clusterv1.Cl
 
 	if err := kubedrain.RunCordonOrUncordon(ctx, drainer, node, true); err != nil {
 		// Machine will be re-reconciled after a cordon failure.
-		log.Error(err, "Cordon failed")
-		return errors.Errorf("unable to cordon node %s: %v", node.Name, err)
+		log.Error(err, "Cordon failed, retry in 20s")
```

Comment:
Should we remove this log or should the following line return `ctrl.Result{RequeueAfter: 20 * time.Second}`?

Reply:
Removed the log